client/my-sites/stats/stats-purchase/stats-purchase-wizard.jsx

```diff
@@ -81,6 +86,8 @@ const ProductCard = ( { siteSlug, siteId, commercialProduct, pwywProduct, redire
 	// change the plan to commercial on the personal plan confirmation
 	const handlePlanSwap = ( e ) => {
 		e.preventDefault();
+		recordTracksEvent( `calypso_stats_plan_switched_from_personal_to_commercial` );
```

Comment:
@Zubie04 asked a good question: Is there a commercial-to-personal equivalent to this event?